client-app/src/BaseAppModel.ts

```diff
@@ -0,0 +1,38 @@
+import {HoistAppModel, XH} from '@xh/hoist/core';
+import {OauthService} from './core/svc/OauthService';
+
+export class BaseAppModel extends HoistAppModel {
+    static override async preAuthAsync() {
+        if (!(await BaseAppModel.shouldSkipOAuth())) {
```

Comment:
might be slightly clearer to have OAuthService do this logical check internally in its init() and become `disabled` and be a no-op.  Situates that common code in a natural place and also avoids the conditional presence of the service.

Reply:
could also just have the app immediately go in to "non-sso" mode the way we did before, if oauth is disabled 

XH.appSpec.isSSO = false;

That way, we don't need any special app in the skip case!